CharacterCreation/03_Feats.txt

```diff
@@ -638,12 +638,20 @@ have no prerequisites.
 
 == Shotgun Expertise  ==
 
-0:	+10m to range R3	
-12:	-15% reload miss chance			
-24:	+5 Gun Reflex				
-36:	-0.5 miss chance @ all ranges			
-48:	+10% perception checks for awareness within 25m.			
-60:	+10% Base damage. If you hit armor with a higher APL than your shotgun, you deal half damage to its AS instead of no damage (and full damage to health if you deplete the AS).
+0:	+5m to range R1. If this would cause R1 to be greater than R2, 
+	also set R2 equal to the new R1.
+12:	May use Weapons - Shotgun in place of Acrobatics for checks to execute 
+	a physical maneuver following violent entry of a room.
+24:	If you succeed a reload check by at least 20, reduce the time taken to 
+	reload by one Action, stacking with other reductions (such as from crits), 
+	to a minimum of one Action.

```

Comment:
I really really like this bonus! Clearly you have a zen-like understanding of what makes shotguns different from the rest and what makes them fun! Nailed it!